### DISCUSSION TITLE
Is it possible to clean up deleted exercises in the overview?

### DISCUSSION THREAD

**Discussion by marcothijs** [dodona-edu/feedback-and-support#22] [Questions - Vragen] [answered]
I have some deleted exercises (in Github) that remain in my overview with an trash can icon in front of it. Is there a possibility to get them out the list. 
Here the links
https://dodona.ugent.be/nl/activities/1420167232/
https://dodona.ugent.be/nl/activities/2139543167/
https://dodona.ugent.be/nl/activities/556579547/

None of them should have submissions.

Thanks.
Marco

**Comment by bmesuere** [comment]
Coincidentally I looked into cleaning up the removed exercises without submissions this week. We can remove them, but this requires a manual intervention on the production system. Therefore, we opted to wait until after the exams in case anything would go wrong. We will then probably also automate this action so it runs once a day.

**Reply by marcothijs** [reply]
Hi, 
Has there been progress on this? 

Thanks
Marco

**Reply by bmesuere** [reply]
Thanks for the reminder. I created an issue to track this request: #4969

**Reply by bmesuere** [reply, marked as answer]
This has now been implemented. Exercises that are deleted for 30 days will be removed in most cases. More information can be found here: https://github.com/dodona-edu/dodona/pull/5095

**Comment by marcothijs** [comment]
Thanks!Marco ThijsOn 13 Nov 2023, at 17:11, Bart Mesuere ***@***.***> wrote:﻿
This has now been implemented. Exercises that are deleted for 30 days will be removed in most cases. More information can be found here: #5095
